recipes-devtools/seeed-python-reterminal/python3-seeed-python-reterminal.bb

```diff
@@ -8,7 +8,7 @@ LIC_FILES_CHKSUM = "file://${WORKDIR}/git/LICENSE;md5=6a91c33fdb9c2f0d4aeab7a112
 
 SRCREV = "${AUTOREV}"
 
-SRC_URI = "git://github.com/Seeed-Studio/Seeed_Python_ReTerminal.git;protocol=git;branch=main \
+SRC_URI = "git://github.com/Seeed-Studio/Seeed_Python_ReTerminal.git;protocol=https;branch=main \
```

Comment:
Any reason for the change from git to https?

Reply:
The reason is Github has changed its security policy, see [its announce](https://github.blog/2021-09-01-improving-git-protocol-security-github/). The same applies to the reterminal-demos commit.

Reply:
Thank you again.
I will merge this change after I tested it.